core/data/data.go

```diff
@@ -95,6 +96,7 @@ func SetupPersistence(file string) error {
 
 	dbBackupTicker := time.NewTicker(1 * time.Hour)
 	go func() {
+                backupFile := filepath.Join(config.BackupDirectory, "owncastdb.bak")
```

Comment:
Some weird spacing here.  If you happen to use VSCode, if you add the Go Language support it'll auto-format your code for you with [gofmt](https://golang.org/cmd/gofmt/).

Reply:
@gabek I should not use the GH editor for these kind of changes. That was a tab vs. space issue. I also removed the trailing spaces. The changes should fix the linter errors.